Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -6,3183 +6,3875 @@
 	objectVersion = 48;
 	objects = {
 
+/* Begin PBXAggregateTarget section */
```

Comment:
@kevinchan159 are all these changes okay? I'm assuming they're just related to updating CocoaPods but I'm curious what will happen when this is merged.

Reply:
yeah I think its just from the cocoapod update so should be fine. i would double check that it runs on your computer before merging though @jackthmp